doc/data/messages/e/empty-docstring/bad.py

```diff
@@ -0,0 +1,2 @@
+def foo():
+    pass  # [emtpy-docstring]
```

Comment:
Not related to the actual MR, but I think code example could be 'real' and 'exciting' with animal, vegetal and real concepts or code example to be more rememberable. Also typing by default would influence begginner to use it later.


```suggestion
def add(a: Any, b: Any) -> Any:
    return a + b  # [emtpy-docstring]
```
Foos and bars are traditional and have the benefit of not introducing more concept than the message itself, but also a little boring and I think human learn better with (fun) examples than with generic examples.

Reply:
You're right about "any", maybe we could another example ? (``def do_nothing() -> None:``).

Reply:
I think you're also right about barebone being a good thing, but I dislike foo/bar for documentation. For a beginner the first thing they have to do is to read https://en.wikipedia.org/wiki/Foobar and learn about our little software programmer internal meme. We can probably make barebone examples that a school children would understand.

Reply:
Ah shit, I completely forgot about this.

I'll keep it in mind for future PRs 👍

Reply:
Don't worry we'll have to review hundred of examples that were in pylint-error, there's no use nitpicking about the style right now 😄